tests/test_task.py

```diff
@@ -1334,6 +1334,17 @@ def test_same_timestamp_and_value(self):
             task_a._task_id_hash, task_b._task_id_hash,
             "task ids should be different since the filenames are different")
 
+    def test_return_value_no_record(self):
+        """TaskGraph: test that `get` raises exception if not set to record."""
+        task_graph = taskgraph.TaskGraph(self.workspace_dir, -1)
+        value_task = task_graph.add_task(
+            func=_noop_function,
+            store_result=False)
+
+        # get wil raise a ValueError because store_result is not True
+        with self.assertRaises(ValueError):

```

Comment:
I've been liking the workflow of testing part of the returned error message just in case another `ValueError` gets mysteriously raised since it is a core Exception class. Could we quickly check against: 
>must set `store_result` to True in `add_task` to invoke this

Reply:
I like it too, I was just lazy. That's why we do a PR! Fixed in 23c910fa6fe686396e80b7c1f2016b60b63fb346